http.go

```diff
@@ -347,8 +337,39 @@ func Get(w http.ResponseWriter, req *http.Request, store mdata.Store, defCache *
 
 	js := bufPool.Get().([]byte)
 	if legacy {
-		sort.Sort(SeriesByTarget(out))
-		js, err = graphiteJSON(js, out)
+		//check for duplicate series names. If found merge the results.
+		seriesByTarget := make(map[string][]Series)
+		for _, series := range out {
+			if _, ok := seriesByTarget[series.Target]; !ok {
+				seriesByTarget[series.Target] = make([]Series, 0)
+			}
+			seriesByTarget[series.Target] = append(seriesByTarget[series.Target], series)
+		}
+		merged := make([]Series, len(seriesByTarget))
+		count := 0
+		for _, series := range seriesByTarget {
```

Comment:
ah ok, never mind then i guess. i would have personally named the variable `idx` or `i` or something instead of `count` since we're not really counting anything with the goal of obtaining a count, the var merely serves the iteration, but whatever, this is a minor point.